src/bridges/uniswap/UniswapBridge.sol

```diff
@@ -155,13 +155,27 @@ contract UniswapBridge is BridgeBase {
         }
     }
 
+    /**
+     * @notice Registers subsidy criteria for a given token pair.
+     * @param _tokenIn - Input token to swap
+     * @param _tokenOut - Output token to swap
+     */
+    function registerSubsidyCriteria(address _tokenIn, address _tokenOut) external {
+        SUBSIDY.setGasUsageAndMinGasPerMinute({
+            _criteria: _computeCriteria(_tokenIn, _tokenOut),
+            _gasUsage: uint32(300000), // 300k gas (Note: this is a gas usage when only 1 split path is used)
```

Comment:
I agree that subsidizing the cheaper case is the right choice to now over-subsidize the bridge and waste assets.